cloudify_cli/commands/plugins.py

```diff
@@ -285,3 +292,135 @@ def set_visibility(plugin_id, visibility, logger, client):
         client.plugins.set_visibility(plugin_id, visibility)
         logger.info('Plugin `{0}` was set to {1}'.format(plugin_id,
                                                          visibility))
+
+
+@plugins.command(name='update',
+                 short_help='Update the plugins of all the deployments of '
+                            'the blueprint [manager only]')
+@cfy.options.blueprint_id(required=True)
+@cfy.options.common_options
+@cfy.options.tenant_name(required=False, resource_name_for_help='plugin')
+@cfy.assert_manager_active()
+@cfy.options.include_logs
+@cfy.options.json_output
+@cfy.pass_logger
+@cfy.pass_client()
+@cfy.options.force(help=helptexts.FORCE_PLUGINS_UPDATE)
+def update(blueprint_id,
+           include_logs,
+           json_output,
+           logger,
+           client,
+           tenant_name,
+           force):
+    """Update the plugins of all the deployments of the given blueprint. This
+    will update the deployments one by one until all succeeded.
+    """
+    utils.explicit_tenant_name_message(tenant_name, logger)
+    logger.info('Updating the plugins of the deployments of the blueprint '
+                '{}'.format(blueprint_id))
+    plugins_update = client.plugins_update.update_plugins(blueprint_id, force)
+    events_logger = get_events_logger(json_output)
+    execution = execution_events_fetcher.wait_for_execution(
+        client,
+        client.executions.get(plugins_update.execution_id),
+        events_handler=events_logger,
+        include_logs=include_logs,
+        timeout=None  # don't timeout ever
```

Comment:
Sure about this? maybe set a long timeout and give the user the option to change it

Reply:
I don't think we should trust the user to give the right timeout because the timeout is dependant on the amount of active deployments there are for that blueprint

Reply:
I think system workflows don't have a timeout regardless